### PR TITLE
Add Krona output for Centrifuge profiles

### PR DESCRIPTION
**Pull request by MillironX**
Follows up on #91

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/visualization_krona.nf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    */
+    ch_input_profiles = profiles
+        .branch {
+            centrifuge: it[0]['tool'] == 'centrifuge'
+            kraken2: it[0]['tool'] == 'kraken2'
+            unknown: true
+        }
+
+    /*
+        Convert Kraken2 formatted reports into Krona text files
```
Comment (jfy133) [inline subworkflows/local/visualization_krona.nf:29]:
```suggestion
        Convert Kraken2 and Centrifuge formatted reports into Krona text files
```

**Review by jfy133 — APPROVED** [review]
One tiny suggestion but :ok_hand: 

Will wait for other PR to go in and then this one!

File: subworkflows/local/visualization_krona.nf (hunk 1 above)
Comment (MillironX) [reply subworkflows/local/visualization_krona.nf:29]: Technically, these aren't reports in Centrifuge format (they've been converted by `CENTRIFUGE_KREPORT`). I can add a note of that, if you'd like.

Comment (jfy133) [reply subworkflows/local/visualization_krona.nf:29]: Yes please! That would be helpful.You can do this in a separate PR though as the tests are passing  now (and take a long time)

**Pull request merged**